src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Sorts the list of all persons in the address book by name in alphabetical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list with index numbers sorted alphabetically by name.\n"
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer (not sure whether this line exceeds or not! :) )